README.md

```diff
@@ -51,6 +51,8 @@ I stumbled across [this quote](http://www.bash.org/?10159):
 
 Nicotine+ does not collect any user data. Different policies may apply to the default Soulseek server, which is not operated by the Nicotine+ Team.
 
+Soulseek is unencrypted, and should not be used for secure communication. Use a VPN if you are concerned about your IP address being public.
```

Comment:
"should not be used for" -> "is not intended for"

It is not warranted to raise any concerns about the IP address, nor is it our remit to suggest using VPN or other such.

Reply:
Soulseek is an unencrypted protocol,...

Reply:
> "should not be used for" -> "is not intended for"
> 
> It is not warranted to raise any concerns about the IP address, nor is it our remit to suggest using VPN or other such.

Right, I suppose you can already draw that conclusion from the statement about encryption.